core/src/ons/util.js

```diff
@@ -20,11 +20,37 @@ import internal from './internal';
 import autoStyle from './autostyle';
 import ModifierUtil from './internal/modifier-util';
 import animationOptionsParse from './animation-options-parser';
+import platform from './platform';
 
 const util = {};
 
 util.globals = {
-  fabOffset: 0
+  fabOffset: 0,
+  supportsPassive: false
+};
+
+platform._runOnActualPlatform(() => util.globals.actualMobileOS = platform.getMobileOS());
+
+try {
+  const opts = Object.defineProperty({}, 'passive', {
+    get() { util.globals.supportsPassive = true; }
+  });
+  window.addEventListener('testPassive', null, opts);
+  window.removeEventListener('testPassive', null, opts);
+} catch (e) { null; }
+
+/**
+ * @param {Element} el Target
+ * @param {String} name Event name
+ * @param {Function} handler Event handler
+ * @param {Object} [opt] Event options (passive, capture...)
+ * @param {Boolean} [isGD] If comes from GestureDetector. Just for testing.
```

Comment:
Do we need to keep this param?

Reply:
@asialgearoid It's internal so I guess it's not an issue :sweat_smile: . It is just for debugging in the future.